scripts/k8s/deploy_kubeflow.sh

```diff
@@ -8,66 +8,52 @@ CONFIG_DIR="${ROOT_DIR}/config"
 # Source common libraries and env variables
 source ${ROOT_DIR}/scripts/common.sh
 
-# Specify credentials for the default user.
-# TODO: Dynamically sed/hash these value into the CONFIG, these are currently not used
-export KUBEFLOW_USER_EMAIL="${KUBEFLOW_USER_EMAIL:-admin@kubeflow.org}"
-export KUBEFLOW_PASSWORD="${KUBEFLOW_PASSWORD:-12341234}"
-
 # Poll for these to be available with the -w flag
-KUBEFLOW_POLL_DEPLOYMENTS="${KUBEFLOW_DEPLOYMENTS:-profiles-deployment notebook-controller-deployment centraldashboard ml-pipeline minio mysql metadata-db jupyter-web-app-deployment katib-mysql}"
+KUBEFLOW_POLL_DEPLOYMENTS="${KUBEFLOW_DEPLOYMENTS:-profiles-deployment notebook-controller-deployment centraldashboard ml-pipeline minio mysql jupyter-web-app-deployment katib-mysql}"
 
-# Speificy how long to poll for Kubeflow to start
+# Specify how long to poll for Kubeflow to start
 export KUBEFLOW_TIMEOUT="${KUBEFLOW_TIMEOUT:-600}"
+export KUBEFLOW_DEPLOY_TIMEOUT="${KUBEFLOW_DEPLOY_TIMEOUT:-1200}"
 
-# Local files/directories to create and place scripts
-export KF_DIR="${KF_DIR:-${CONFIG_DIR}/kubeflow-install}"
-export KFCTL="${KFCTL:-${CONFIG_DIR}/kfctl}"
-export KUSTOMIZE="${KUSTOMIZE:-${CONFIG_DIR}/kustomize}"
-export KUBEFLOW_DEL_SCRIPT="${KF_DIR}/deepops-delete-kubeflow.sh"
-
-export KUBEFLOW_MPI_DIR="${KUBEFLOW_MPI_DIR:-${KF_DIR}/mpi}"
-export KUBEFLOW_MPI_MANIFESTS_REPO="${KUBEFLOW_MPI_MANIFESTS_REPO:-https://github.com/kubeflow/manifests}"
+# Define Kubeflow manifests location
+export KUBEFLOW_MANIFESTS_DEST="${KUBEFLOW_MANIFESTS_DEST:-${CONFIG_DIR}/kubeflow-install/manifests}"
+export KUBEFLOW_MANIFESTS_URL="${KUBEFLOW_MANIFESTS_URL:-https://github.com/kubeflow/manifests}"
+export KUBEFLOW_MANIFESTS_VERSION="${KUBEFLOW_MANIFESTS_VERSION:-v1.4.1}"
```

Comment:
I had been using hashes instead of branch names because they tend to change the files in the release branch. But that kept breaking anyways, so I'm good with this.